modules/core/shared/src/main/scala/lucuma/core/math/parser/Epoch.scala

```diff
@@ -30,5 +31,15 @@ trait EpochParsers {
       }
       .named("epoch")
 
+  /** Parser for an `Epoch` with a default Julian schemme. */
+  val epochLenient: Parser[Epoch] =
```

Comment:
Can we put "Julian" in the name somewhere? `julianEpoch` maybe?

Reply:
Sure, is the name used in error messages?